src/RecoveryServices/RecoveryServices.Backup.Test/ScenarioTests/IaasVm/ItemTests.cs

```diff
@@ -75,7 +90,11 @@ public void TestAzureVMFullRestore()
                 _logger, PsBackupProviderTypes.IaasVm, "Test-AzureVMFullRestore");
         }
 
+#if NETSTANDARD
+        [Fact(Skip = "Needs investigation, TestManagementClientHelper class wasn't initialized with the ResourceManagementClient client.")]
```

Comment:
The test that is failing for a particular platform was marked to be skipped in a PR by @MiYanni (https://github.com/Azure/azure-powershell/pull/7497/files#diff-5ac3bb75515258487eb34f862e7890dd) which I removed in this PR. Moreover, it looks like a platform issue which was solved later in dotnet core 2.1 (https://stackoverflow.com/questions/48719442/dotnet-core-2-0-on-macos-gets-libcurl-and-ssl-error-triggered-by-httpclient-when)

Reply:
@markcowl We should discuss running tests as netcore 2.1 instead of 2.0, sooner rather than later. I'm sure there are other fixes like this one above.